### DISCUSSION TITLE
ReCaptcha per event

### DISCUSSION THREAD

**Discussion by BudoAsahi** [liedekef/events-made-easy#578] [Q&A]
Hi Franky, 
I noticed I can't un-check the recaptcha per event. When I create or update an event without recaptcha (+ no recaptcha in the form-template), it gets switched on automatically. I have to change it in the database to shut it off. 
It worked okay before (until ?). Is it a bug or me being confused?

**Comment by liedekef** [comment]
Confirmed as a bug and fixed here:
https://github.com/liedekef/events-made-easy/commit/7bbdfcb0b00622b318d34094f306b69bdd905df9

**Reply by BudoAsahi** [reply]
Thank you!
